.github/workflows/repo-sync.yml

```diff
@@ -0,0 +1,27 @@
+name: Repo Sync
+
+on:
+  schedule: 
+  - cron: "*/30 06 * * *" # every 15 minutes. set to whatever interval you like
```

Comment:
could you remove the comment or correct it to match the actual schedule in case somebody wants to modify this later?